### PR TITLE
Feature/om 393

### PR DESCRIPTION
**Pull request by Citronhariom**
Description : Build has been completed for the OM-393

### CODE REVIEW

File: force-app/main/default/classes/OpportunityValidationWrapperTest.cls [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+@isTest
+public with sharing class OpportunityValidationWrapperTest {
+
+    static testMethod void validateOpportunitiesBeforeInsertMethod() {
+        
+        test.startTest();
+        Group testGroup = new Group(Name = 'Customer_Success', type = 'Queue');
+        insert testGroup;
+        QueueSObject mappingObject = new QueueSobject(QueueId = testGroup.Id, SObjectType = 'Case');
+        System.runAs(new User(Id = UserInfo.getUserId())) {
```
Comment (MHigel) [inline force-app/main/default/classes/OpportunityValidationWrapperTest.cls:12]: This code will always execute as the running user, so using `System.runAs` here doesn't change the test's behavior.

File: force-app/main/default/classes/OpportunityValidationWrapperTest.cls [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+        update ocr;
+       
+        // Opportunity update 
+        opp.Amount = 1.0;
+        opp.StageName = 'Proposal';
+        update opp;
+        opp1.Amount = 1.0;
+        update opp1;
+        
+        test.stopTest();
```
Comment (MHigel) [inline force-app/main/default/classes/OpportunityValidationWrapperTest.cls:89]: This test has no asserts to validate the output.

**Review by MHigel — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/OpportunityValidationWrapperTest.cls (hunk 2 above)
Comment (MHigel) [reply force-app/main/default/classes/OpportunityValidationWrapperTest.cls:89]: @Citron-mohammad-Rafi this comment is still outstanding.

**Review by MHigel — CHANGES_REQUESTED** [review]

File: force-app/main/default/classes/OpportunityValidationWrapperTest.cls (hunk 2 above)
Comment (Citronhariom) [reply force-app/main/default/classes/OpportunityValidationWrapperTest.cls]: Hi @MHigel,
I have updated My PR , Please review it once and let me know if any changes required  .

**Review by MHigel — CHANGES_REQUESTED** [review]

**Pull request closed without merging**